packages/manager/modules/backup-agent/src/components/Fields/index.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useId } from 'react';
+
+import { Control, FieldValues, Path, UseControllerProps, useController } from 'react-hook-form';
+
+import { OdsComboboxItem, OdsFormField, OdsText } from '@ovhcloud/ods-components/react';
+
+import { RhfCombobox } from '@/components/Fields/RhfCombobox.component';
+
+import { RhfFieldContext, RhfFieldContextParams, useRhfFieldContext } from './RhfField.context';
+import { RhfInput } from './RhfInput.component';
+import { RhfQuantity } from './RhfQuantity.component';
+
+type RhfFieldProps<
+  TFieldValues extends FieldValues,
+  TName extends Path<TFieldValues>,
+> = React.ComponentProps<typeof OdsFormField> & {
+  controllerParams: UseControllerProps<TFieldValues, TName>;
+  helperMessage?: string;
+  isHiddenError?: boolean;
+  control: Control<TFieldValues, TName>;
+};
+
+export const RhfField = <TFieldValues extends FieldValues, TName extends Path<TFieldValues>>({
+  controllerParams,
+  className,
+  helperMessage,
+  isHiddenError,
+  control,
+  ...rest
+}: Readonly<RhfFieldProps<TFieldValues, TName>>) => {
```

Comment:
I see that you call you component like this:

```tsx
      <RhfField
        control={control}
        data-testid="rhfField"
        controllerParams={register('inputTest')}
        helperMessage={HELPER_MESSAGE}
      >
```

But you don't need to (and shouldn't) use `register` and 'control` together, they are both intended to register components, but in different ways.
I you want to be able to call `usController`, you just need `control` and `name`